src/nanodbc.h

```diff
@@ -1019,6 +1019,10 @@ class connection
     /// \brief Returns the native ODBC environment handle.
     void* native_env_handle() const;
 
+
+    /// \brief Returns information from the ODBC connection as a string.
+    string_type get_string_info(short info_type) const;
```

Comment:
Unless there is a plan to add `get_integer_info` or similar and `get_string_info` is the only function to available, lack of coercion is not relevant. IMHO, `get_string_info` is not a good but ambiguous  name: it does not return an info details about a string (compare with `get_host_info`).  What the function does is: `get_info_as_string`. I may sound anal about naming, but IMHO such details are fairly important for self-descriptive and discoverable interface.